### DISCUSSION TITLE
Alerting Rule expand button hidden until search term is entered

### DISCUSSION THREAD

**Discussion by wjordan** [grafana/grafana#56574] [Alerting]
(Converted from [issue](https://github.com/grafana/grafana/issues/56573) as [requested](https://github.com/grafana/grafana/issues/56573#issuecomment-1271891382))

The expand button appears and disappears when a search term is entered, resulting in a rather jarring UX:

![alert-filter](https://user-images.githubusercontent.com/56541/194615839-0f6620e8-15eb-4257-b3fe-1a5aec9e4091.gif)

It appears that this design choice was [intentional](https://github.com/grafana/grafana/pull/38770#discussion_r713848852) to indirectly prevent expanding a large number of alerts, but a disappearing button ends up being rather confusing and distracting. If anything, the button should simply be `disabled` (greyed-out and unclickable) when the number of displayed alert rules is greater than the amount that can reasonably handled without freezing.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
